modules/apps/asset/asset-browser-web/src/main/java/com/liferay/asset/browser/web/internal/item/selector/AssetEntryItemSelectorView.java

```diff
@@ -119,6 +119,14 @@ private DynamicServletRequest _getDynamicServletRequest(
 				"multipleSelection",
 				_toStringArray(
 					!assetEntryItemSelectorCriterion.isSingleSelect())
+			).put(
+				"scopeGroupClassPK",
+				_toStringArray(
+					assetEntryItemSelectorCriterion.getScopeGroupClassPK())
+			).put(
+				"scopeGroupType",
+				_toStringArray(
+					assetEntryItemSelectorCriterion.getScopeGroupType())
 			).put(
```

Comment:
Step 2: From the `ItemSelectorCriterion`, add the scope's type and classPK as attributes to the request.